client/my-sites/media-library/list.jsx

```diff
@@ -149,18 +112,35 @@ export const MediaLibraryList = React.createClass( {
 		}
 	},
 
-	getItemRef: function( item ) {
-		return 'item-' + item.ID;
-	},
+	renderItem: function( { rowIndex, columnIndex, key, style } ) {
+		const index = rowIndex * this._columnCount + columnIndex;
+		const item = this._gridItems[ index ];
 
-	renderItem: function( item ) {
-		var index = findIndex( this.props.media, { ID: item.ID } ),
-			selectedItems = this.props.mediaLibrarySelectedItems,
-			selectedIndex = findIndex( selectedItems, { ID: item.ID } ),
-			ref = this.getItemRef( item ),
-			showGalleryHelp;
+		style = {
+			...style,
+			fontSize: this.props.mediaScale * 225,
+			top: style.top + this.props.padding,
+			left: style.left + this.props.padding,
+			width: style.width - this.props.padding * 2,
+			height: style.height - this.props.padding * 2,
+		};
 
-		showGalleryHelp = (
+		if ( ! item ) {
+			return;
+		}
+
+		if ( item.loading ) {
```

Comment:
not sure if it applies here but `react-virtualized` also provides the `noContentRenderer` prop for when the list/grid is empty